pkg/monitor/controller_test.go

```diff
@@ -0,0 +1,635 @@
+		assert.Nil(t, controller.controllerMonitor.controllerCRD)
+	})
+	t.Run("create-success", func(t *testing.T) {
+		clientset := fakeclientset.NewSimpleClientset()
+		controller := newControllerMonitor(clientset)
+		initController(controller)
+		controller.controllerMonitor.querier.GetControllerInfo(newCRD, false)
+		controller.controllerMonitor.syncControllerCRD()
+		crd, err := controller.controllerMonitor.client.CrdV1beta1().AntreaControllerInfos().Get(context.TODO(), crdName, metav1.GetOptions{})
+		newCRD.ControllerConditions[0].LastHeartbeatTime.Time = crd.ControllerConditions[0].LastHeartbeatTime.Time
+		assert.NoError(t, err)
+		assert.Equal(t, newCRD, crd)
+	})
+	t.Run("create-failure", func(t *testing.T) {
+		clientset := fakeclientset.NewSimpleClientset()
+		clientset.PrependReactor("create", "antreacontrollerinfos", func(action cgtesting.Action) (handled bool, ret runtime.Object, err error) {
+			return true, &v1beta1.AntreaControllerInfo{}, errors.New("error in creating controller crd")
+		})
+		controller := newControllerMonitor(clientset)
+		initController(controller)
+		controller.controllerMonitor.syncControllerCRD()
+		_, err := controller.controllerMonitor.createControllerCRD(crdName)
+		assert.Equal(t, "error in creating controller crd", err.Error())
+		assert.Nil(t, controller.controllerMonitor.controllerCRD)
+	})
+	t.Run("entire update-success", func(t *testing.T) {
+		clientset := fakeclientset.NewSimpleClientset(existingCRD)
+		controller := newControllerMonitor(clientset)
+		initController(controller)
+		controller.controllerMonitor.querier.GetControllerInfo(entirelyUpdatedCRD, false)
+		controller.controllerMonitor.syncControllerCRD()
+		crd, err := controller.controllerMonitor.client.CrdV1beta1().AntreaControllerInfos().Get(context.TODO(), crdName, metav1.GetOptions{})
+		assert.NoError(t, err)
+		entirelyUpdatedCRD.ControllerConditions[0].LastHeartbeatTime.Time = crd.ControllerConditions[0].LastHeartbeatTime.Time
+		assert.Equal(t, entirelyUpdatedCRD, crd)
+	})
+	t.Run("entire update-failure", func(t *testing.T) {
+		clientset := fakeclientset.NewSimpleClientset(existingCRD)
+		clientset.PrependReactor("update", "antreacontrollerinfos", func(action cgtesting.Action) (handled bool, ret runtime.Object, err error) {
+			return true, &v1beta1.AntreaControllerInfo{}, errors.New("error updating controller crd")
+		})
+		controller := newControllerMonitor(clientset)
+		initController(controller)
+		controller.controllerMonitor.syncControllerCRD()
+		_, err := controller.controllerMonitor.client.CrdV1beta1().AntreaControllerInfos().Update(context.TODO(), existingCRD, metav1.UpdateOptions{})
+		assert.Equal(t, "error updating controller crd", err.Error())
+		assert.Nil(t, controller.controllerMonitor.controllerCRD)
+	})
+}
+
+func TestEnqueueNode(t *testing.T) {
+	node := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{Name: "TestNode", Namespace: "ns1", Labels: map[string]string{"en": "vm2"}},
+	}
```

Comment:
Better to use following format, please check other occurrences.
```suggestion
	node := &v1.Node{
		ObjectMeta: metav1.ObjectMeta{
		Name: "TestNode", 
		Namespace: "ns1", 
		Labels: map[string]string{"en": "vm2"},
		},
	}
```